src/routes/index.svelte

```diff
@@ -37,12 +37,47 @@
 	async function updateEntries() {
 		fetchIndex += 1
 		let thisFetchIndex = fetchIndex
-		const res = await fetch(
-			`/api/entries.json?visible=${showVisible}&unlisted=${showUnlisted}&hidden=${showHidden}`
-		)
+		let search_value = (document.getElementById('search') as HTMLInputElement).value || null
```

Comment:
This should be in the global scope and binded to the searchbox similar to how showVisible/showUnlisted/showHidden are done (except with bind:value instead of bind:checked)